centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
I don't think the server sends back only txt files. I'm pretty sure the server is not sensitive to extensions, it just happens that a) our client is sensitive to extensions, even though I thought we fixed this months ago and b) our input files happen to have the .txt extension. The cit lab already had to work around this, so it's not worth changing unless we can accommodate their current workflow. It doesn't make sense to merge a "bug fix" that causes tons of effort by forcing us to change all of the extensions on the backend. It saves way more time to make the client extension agnostic